ansible_collections/arista/avd/roles/eos_designs/docs/tables/management-flow-tracking-settings.md

```diff
@@ -128,6 +131,13 @@
         # Flow tracker name as defined in flow_tracking_settings.
         name: <str>
 
+      # Enable flow-tracking on all non uplink WAN HA links..
```

Comment:
Should we not just call it "Direct WAN HA" links all over?

Reply:
done